typescript/sdk/src/token/EvmERC20WarpModule.ts

```diff
@@ -74,7 +81,64 @@ export class EvmERC20WarpModule extends HyperlaneModule<
     TokenRouterConfigSchema.parse(expectedConfig);
     const actualConfig = await this.read();
 
-    return this.updateIsm(actualConfig, expectedConfig);
+    const transactions = [];
+
+    transactions.push(
+      ...(await this.updateIsm(actualConfig, expectedConfig)),
+      ...(await this.updateRemoteRouters(actualConfig, expectedConfig)),
+    );
+
+    return transactions;
+  }
+
+  /**
+   * Updates the remote routers for the Warp Route contract.
+   *
+   * @param actualConfig - The on-chain router configuration, including the remoteRouters array.
+   * @param expectedConfig - The expected token router configuration.
+   * @returns A array with a single Ethereum transaction that need to be executed to enroll the routers
+   */
+  async updateRemoteRouters(
+    actualConfig: TokenRouterConfig,
+    expectedConfig: TokenRouterConfig,
+  ): Promise<AnnotatedEV5Transaction[]> {
+    const updateTransactions: AnnotatedEV5Transaction[] = [];
+
+    if (expectedConfig.remoteRouters) {
```

Comment:
might increase readability to short-circuit with the if rather than punch everything out a tab, like
```suggestion
    if (!expectedConfig.remoteRouters) {
        logger.trace('updateRemoteRouters(): Nothing to update: expectedConfig.remoteRouters undefined.');
        return [];
    }
    
    // remaining code block here
```

Reply:
+1

Reply:
Fixed